arm-software/linux/docs/OpenMPSupport.md

```diff
@@ -0,0 +1,87 @@
+<!--===- docs/OpenMPSupport.md
+
+   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Is this actually part of the LLVM Project?

Reply:
This directory is outside llvm-project so will remove the banner.